tutorials/express/post-form.md

```diff
@@ -0,0 +1,27 @@
+Using the `FormData()` class, you can send form data to your express backend. <br />
+Frontend:
+
+```javascript
+async function run() {
```

Comment:
Just reuse the examples from this tutorial: https://github.com/mastering-js/masteringjs.io/blob/master/tutorials/axios/form-data.md

People generally use FormData to send files, so best to provide an example of sending a file.